Sources/PointFree/Account/Invite.swift

```diff
@@ -261,42 +261,26 @@ private func validateIsNot(currentUser: User) -> (User) -> EitherIO<Error, User>
   }
 }
 
-private func redirectCurrentSubscribers<A, B>(
-  _ middleware: @escaping M<T3<A, User?, B>>
-) -> M<T3<A, User?, B>> {
-  @Dependency(\.database) var database
-  @Dependency(\.stripe) var stripe
+func redirectCurrentSubscribers<A>(
+  _ middleware: @escaping M<A>
+) -> M<A> {
+  @Dependency(\.subscriberState) var subscriberState
```

Comment:
I think this helper was written before we had access to `subscriberState` in the dependency system, and so now the logic can be greatly simplified. No need to fetch subscriptions for the database/stripe since that has already been done for us.